console/backend/src/main/java/org/frankframework/console/controllers/UpdateLoggingConfig.java

```diff
@@ -73,6 +74,20 @@ public ResponseEntity<?> getLogDefinitions(@RequestParam(value = "filter", requi
 		return callSyncGateway(request);
 	}
 
+	@RolesAllowed({"IbisDataAdmin", "IbisAdmin", "IbisTester"})
+	@Relation("logging")
+	@Description("create a new logger definition")
+	@PostMapping(value = "/server/logging/settings", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+	public ResponseEntity<?> createLogDefinition(LogDefinitionMultipartBody multipartBody) {
+		String logger = RequestUtils.resolveRequiredProperty("logger", multipartBody.logger(), null);
+		String level = RequestUtils.resolveRequiredProperty("level", multipartBody.level(), null);
+
+		RequestMessageBuilder request = RequestMessageBuilder.create(BusTopic.LOG_DEFINITIONS, BusAction.UPLOAD);
+		request.addHeader("logPackage", logger);
```

Comment:
Heel eigenlijk is het de loggernaam, je zou natuurlijk ook een log-group kunnen pakken, en meestal is dit de packagenaam (bij applicaties die classname als logger gebruiken). Sommige applicaties zoals Liquibase gebruiken `liquibase.random-naampje` ongerelateerd aan een klasse. Super naar debuggen maar het kan dus ook op deze unieke manier. Wat hiervoor een goede naam is weet ik niet dus bij deze maar lekker laten. Maar dat weet je van deze context af.